.github/workflows/provenance.yaml

```diff
@@ -60,6 +60,12 @@ jobs:
           ./oak_functions/loader/bin/oak_functions_loader \
           ./scripts/runner build-functions-server
 
+      - name: Validate the shape of the generated provenance files
```

Comment:
Could this, instead of being a step that could fail, be a condition for adding provenances to the provenance branch  in the next step?

Reply:
Possibly, yes. :) I'm not sure if silent failures are desire-able though

Reply:
I would not call that a silent failure. The next step could still create a comment on the PR reporting that no provenances were added to the provenance branch, if the condition fails, similar to what we currently have for reproducibility index. We'd still be notified, but not as a failed build step. Actually I think leaving a comment on the PR is better than a failing post-merge step. Comments are more permanent and can give a more specific description of the situation than a failed post-merge build, at least on GitHub.